src/races/services/races.service.ts

```diff
@@ -87,35 +83,74 @@ export class RacesService extends createPrismaBase(MODELS.Race) {
     const nextYear = format(addYears(startOfYear(new Date()), 2), 'M d, yyyy')
 
     const now = format(new Date(), 'M d, yyyy')
-    const query = {
+    const query: Prisma.RaceWhereInput = {
       state: state.toUpperCase(),
       positionSlug,
       electionDate: {
         gte: new Date(now),
         lt: new Date(nextYear),
       },
-    } as RaceQuery
+    }
 
     if (city && cityRecord) {
       query.municipalityId = cityRecord.id
     } else if (countyRecord) {
       query.countyId = countyRecord.id
     }
-    const race = (await this.findFirst({
+    const races = await this.findMany({
       where: query,
       orderBy: {
         electionDate: 'asc',
       },
-    })) as Race
+      include: {
+        municipality: true,
+        county: true,
+      },
+    })
 
-    if (!race) {
+    if (races.length === 0) {
       return null
     }
 
-    race.municipality = cityRecord
-    race.county = countyRecord
+    const race = races[0]
+    const positions: Array<string | undefined> = []
+    for (let i = 0; i < races.length; i++) {
+      positions.push(races[i].data.position_name)
+    }
```

Comment:
Guessing this is just copy/pasted from tgp.. but this _could_ just be a `.map()` if we wanna do a bit of cleanup.